static/js/dashboard.js

```diff
@@ -317,6 +318,88 @@ function switchFrom(el1, el2, el3, el4) {
 
 var firstSwitch = true
 
+function initValidatorCountdown(validatorIndex, queueId, ts) {
+  var now = Math.round(new Date().getTime() / 1000)
+  var secondsLeft = ts - now
+  var seconds = secondsLeft % 60
+  var min = ((secondsLeft - seconds) / 60) % 60
+  var hour = Math.round((secondsLeft - seconds - min * 60) / 3600) % 24
+  var days = Math.round((secondsLeft - seconds - min * 60 - hour * 60 * 60) / 86400)
+  setValidatorCountdown(validatorIndex, queueId, days, hour, min, seconds)
+
+  if (!countdownIntervals.has(validatorIndex)) {
+    countdownIntervals.set(
+      validatorIndex,
+      setInterval(function () {
+        if (hour <= 0 && min <= 0 && seconds <= 0 && days <= 0) {
+          clearInterval(countdownIntervals.get(validatorIndex))
```

Comment:
Not sure if its nice to just remove it (if it was displayed before) from a user experience... maybe write some text "Your Validator should be active any moment now" ... maybe trigger a refresh of the data or add a 'refresh' button to the tooltip?

Reply:
I don't remove the tooltip I just stop the interval since there is nothing to update if it shows all times as 0.

I implemented this like the validator page does it (except for the rocket). When you reach a time of 0 it will just remain like that. Even if you reload the page it is not immediately active since all it means is that "We have reached the epoch where the validator will be activated". It can still take a few minutes to become active.

But I agree that it looks weird when you approach the end of the time for a few reasons (e.g. a few validators can all be #1 in the queue since they will all be handled in the same epoch) so I will discuss that in the BIDS.

Reply:
As written in the BIDS showing “This validator is currently #1 in Queue” with zero time left is fine.